py3dtilers/Common/object_to_tile.py

```diff
@@ -108,21 +123,44 @@ def get_centroid(self):
                 centroid[1] / len(self),
                 centroid[2] / len(self)]
 
-    def translate_tileset(self,offset):
+    def translate_tileset(self, offset):
```

Comment:
Many differences in this PR are coding style related. Yet there seems to be some flake8 hookups. How do those two info reconcile ?

Reply:
I applied the flake8 standard only on what I worked on (to avoid merge conflicts). Now, only the Obj and Ifc tilers aren't following the standard, maybe @clementcolin should do it ?